Templates/HttpTrigger-FSharp/run.fsx

```diff
@@ -1,16 +1,30 @@
 #r "System.Net.Http"
+#r "Newtonsoft.Json"
 
 open System.Net
 open System.Net.Http
+open Newtonsoft.Json
+
+type Named = {
+    name: string
+}
 
 let Run(req: HttpRequestMessage, log: TraceWriter) =
-    log.Info(sprintf 
-        "F# HTTP trigger function processed a request.")
+    async {
+        log.Info(sprintf 
+            "F# HTTP trigger function processed a request.")
```

Comment:
Not sure how useful these log messages are, though they're probably the same for other languages @soninaren ? If we're writing a log message, we should output some additional details from the request, or remove the messages. Thoughts?